client/src/utils/utils.js

```diff
@@ -14,5 +20,6 @@ const getLocalStorageCart = (cartId) => {
   return cart;
 };
 
+exports.isoDateConverter = isoDateConverter;
```

Comment:
Strange to mix CommonJS and ES modules 🤔  As React uses ES mods maybe stick to those